cocos/2d/CCLabel.cpp

```diff
@@ -1056,11 +1056,11 @@ void Label::enableGlow(const Color4B& glowColor)
     }
 }
 
-void Label::enableOutline(const Color4B& outlineColor,int outlineSize /* = -1 */)
+void Label::enableOutline(const Color4B& outlineColor,float outlineSize /* = -1 */)
```

Comment:
Hi @newnon, did you get any reason that outlineSize should be `float` instead of `int`?